reframe/core/launchers/mpi.py

```diff
@@ -124,3 +124,26 @@ def command(self, job):
             ret.append(opt)
 
         return ret
+
+
+@register_launcher('lrun')
+class LrunLauncher(JobLauncher):
+    '''LLNL's custom parallel job launcher.'''
+    def command(self, job):
+        num_tasks = job.num_tasks or 1
+        num_tasks_per_node = job.num_tasks_per_node or 1
+        num_nodes = int(num_tasks // num_tasks_per_node)
```

Comment:
The `//` is by definition the integer division and both operands are guaranteed to be integers:

```suggestion
        num_nodes = num_tasks // num_tasks_per_node
```

Reply:
done